subsys/mgmt/mcumgr/grp/zephyr_basic/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021-2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Statistics management group public API is exposed by MCUmgr API
+# interface, when Statistics management is enabled.
```

Comment:
Don't think this is relevant to this file

Reply:
Yes, fixed.